packages/core/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "devDependencies": {
     "@types/json-stable-stringify": "^1.0.34",
+    "@types/merge-deep": "^3.0.1",
```

Comment:
is there a reason we went for v3.0.1 instead of 3.x or v4

Reply:
ah since `lavamoat-core`'s on v3.0.x

```console
➜  LavaMoat git:(boneskull/more-test-types) npm list merge-deep
lavamoat-monorepo@0.0.0 /Users/leo/Documents/GitHub/LavaMoat
└─┬ lavamoat-core@15.0.0 -> ./packages/core
  └── merge-deep@3.0.3
```

Reply:
oops was looking at the [wrong](https://github.com/TehShrike/deepmerge) lib earlier (3.x/v4 don't exist for ours)

so 3.0.3 looks correct here? to align w our pkg

```suggestion
    "@types/merge-deep": "^3.0.3",
```